working-groups/user-interface/About.md

```diff
@@ -0,0 +1,44 @@
+# User-interface (UI) working group
+Hello and welcome. This page talks about the UI working group. If you have any queries about the UI working group after going through this page please feel free to reach out to us on [slack.](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw)
```

Comment:
`This document outlines all the information about the User-interface (UI) working group. For any queries about the UI working group, please reach out to us [Slack](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw)`.